Microsoft.Toolkit.Uwp.UI.Controls/TabbedCommandBar/TabbedCommandBar.xaml

```diff
@@ -0,0 +1,1033 @@
+                                        <SolidColorBrush x:Key="TopNavigationViewItemForeground"
+                                                         Color="{ThemeResource SystemAccentColor}" />
+                                        <SolidColorBrush x:Key="TopNavigationViewItemForegroundSelected"
+                                                         Color="{ThemeResource SystemAccentColor}" />
+                                        <SolidColorBrush x:Key="TopNavigationViewItemForegroundPointerOver"
+                                                         Color="{ThemeResource SystemAccentColorLight2}" />
+                                        <SolidColorBrush x:Key="TopNavigationViewItemForegroundPressed"
+                                                         Color="{ThemeResource SystemAccentColorLight2}" />
+                                    </NavigationViewItem.Resources>
+                                </NavigationViewItem>
+                            </DataTemplate>
+                            <controls:TabbedCommandBarItemTemplateSelector x:Key="DefaultTabbedCommandBarItemTemplateSelector"
+                                                                           Contextual="{StaticResource ContextualTabTemplate}"
+                                                                           Normal="{StaticResource NormalTabTemplate}" />
+                        </Grid.Resources>
+
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="Auto" />
+                            <RowDefinition Height="Auto" />
+                        </Grid.RowDefinitions>
+
+                        <NavigationView x:Name="PART_RibbonNavigationView"
+                                        Background="{TemplateBinding Background}"
+                                        MenuItemTemplateSelector="{StaticResource DefaultTabbedCommandBarItemTemplateSelector}"
+                                        PaneDisplayMode="Top"
+                                        Style="{StaticResource RibbonTabNavigationViewStyle}" />
+
+                        <Border Grid.Row="1"
+                                HorizontalAlignment="Stretch"
+                                Background="{ThemeResource TabbedCommandBarContentBackground}" />
+                        <ContentControl x:Name="PART_RibbonContent"
+                                        Grid.Row="1"
+                                        HorizontalAlignment="Stretch"
+                                        HorizontalContentAlignment="Stretch">
+                            <ContentControl.Resources>
+                                <Storyboard x:Name="TabChangedStoryboard">
+                                    <PopInThemeAnimation TargetName="PART_RibbonContent" />
+                                </Storyboard>
+                                <Style TargetType="ComboBox" />
+                                <!--<Style TargetType="ComboBox">
```

Comment:
Any reason to keep these big comment blocks?

Reply:
The reason for this one (the ComboBox style) is because there's currently a bug that causes an unhandled native exception in WUX when the ComboBox style is overridden here. I've yet to make a minimum repo for it. Anyway, that style is supposed to make ComboBoxes look presentable in the primary commands space and also play nicely with overflow (because ATM they're completely unusable once they get put into the overflow menu).

Presumably that style should be placed in its own file and given a key (like ComboBoxAppBarStyle)?

Reply:
Also, I don't want to make too many changes to the ComboBox style. Is there any way to change parts of the template while leaving other parts of it as the default, or do I have to retemplate the entire control?